apps/modernization-ui/src/apps/landing/About/About.tsx

```diff
@@ -46,8 +46,8 @@ export const About = () => {
                 </a>
             </p>
             <p>
-                If you cannot access the Qualtrics link, please reach out to us at
-                <a href="mailto:nbs@cdc.gov">nbs@cdc.gov</a>, and we will work with you on an alternate method for
+                If you cannot access the link, please reach out to us at
+                <a href="mailto:nbs@cdc.gov"> nbs@cdc.gov</a>, and we will work with you on an alternate method for
```

Comment:
suggestion: The other links are using `{' '}` to insert the space before the links, can this use that also?

Reply:
So for this one because prettier keeps correcting me and wants to adjust the formatting it automatically removes it each time I format the document so it basically corrected it like so